src/content/docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic.mdx

```diff
@@ -146,3 +146,9 @@ DevOps, site-reliability, and network operation teams need reliable, real-time a
 * Let [Applied Intelligence](/docs/new-relic-one/use-new-relic-one/new-relic-ai/introduction-new-relic-ai), our hybrid machine learning engine, automatically detect anomalies, reduce alert noise, and enrich incidents with context so that you can respond faster to incidents.
 * [Proactive detection](/docs/new-relic-one/use-new-relic-one/new-relic-ai/proactive-detection-new-relic-ai): Be notified of unusual app behavior and get an analysis of this unusual behavior sent to Slack. Not using Slack? Set up a webhook to deliver messages when you need them.
 * [Get notifications:](/docs/alerts/new-relic-alerts/getting-started/introduction-new-relic-alerts) Set up alerts across your data sources and get notified when systems need your attention. Preserve your attention and control how many threshold violations should fire before you're notified.
+
+## Troubleshoot from anywhere in your stack [#start-anywhere]
+
+Being fully-connected, the New Relic UI allows you to start your observability journey from any element of your stack. For example, you can get to crucial infrastructure logs from traces of an application running on a problematic Kubernetes pod.
+
+Use the [Explorer](/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts/) in [New Relic One](https://one.newrelic.com/) to access and observe the full stack of your software, see performance data and alerting status at a glance, and check relationships. We provide you with a simple, yet powerful visual tool to monitor all your [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/), that is, anything we can identify that reports data. In the New Relic ecosystem, entities include basic components like applications, hosts, containers, or database services, but it can also refer to custom groupings of such elements. You can also [create your own entities](https://github.com/newrelic/entity-definitions#entity-definitions). The more entities you instrument, the more data you'll bring in. The more data you've brought to New Relic, the more you'll understand your metrics, events, logs, and traces.
```

Comment:
no need for comma after "simple, yet"

Reply:
"In the New Relic ecosystem, entities include" is plural so change "it" in same sentence to "they"

Reply:
Done. Good catches!